Doc/library/crypt.rst

```diff
@@ -125,14 +125,18 @@ The :mod:`crypt` module defines the following functions:
    The return value is a string suitable for passing as the *salt* argument
    to :func:`crypt`.
 
-   *log_rounds* specifies the binary logarithm of the number of rounds
-   for ``crypt.METHOD_BLOWFISH``, and is ignored otherwise.  ``8`` specifies
-   ``256`` rounds.
+   *rounds* specifies the number of rounds for ``METHOD_SHA256``,
+   ``METHOD_SHA512`` and ``METHOD_BLOWFISH``, and is ignored otherwise.
+   For ``METHOD_SHA256`` and ``METHOD_SHA512`` it must be an integer between
+   ``1000`` and ``999999999``.  For ``METHOD_BLOWFISH`` it must be a power of
+   two between ``16`` and ``2147483648`` (2\ :sup:`32`), the default is
+   ``4096``.  If it isn't a power of two, it will be rounded up to the next
```

Comment:
Can we use "_" in integer literals to make them more readable?

999999999: 9_999_99_999
2147483648: 2_147_483_648